internal/node/node.bzl

```diff
@@ -637,6 +638,13 @@ nodejs_binary = rule(
     ],
 )
 
+def nodejs_binary_macro(name, **kwargs):
+    nodejs_binary(
+        name = name,
+        entry_point = maybe_directory_file_path(name, kwargs.pop("entry_point", None)),
```

Comment:
it's annoying that the stardoc generation now cannot take this macro into account. The simplest thing is to document in entry_point "even though we said it's a Label, surprise! it can also be a dict"
The hardest thing is to be like `ts_project` and move all the docs to the macro.